addons_daily/utils/helpers.py

```diff
@@ -65,7 +65,7 @@ def load_raw_pings(sc, date):
         .where(docType="main")
         .where(appUpdateChannel="release")
         .where(submissionDate=date)
-        .records(sc)
+        .records(sc, sample=0.01)
```

Comment:
let's keep this to a 1% sample too, its not worth the compute power to leave unsampled.